source/agora/test/Base.d

```diff
@@ -84,6 +84,20 @@ public class TestNetwork : Network
         return api;
     }
 
+    /***************************************************************************
+
+        Start each of the nodes
+
+        Params:
+            count = Expected number of nodes
```

Comment:
What ? :D

Reply:
This was moved from one place to another. And also, I remove it in a separate commit!